src/components/swap/components/transaction-review/index.tsx

```diff
@@ -58,30 +60,24 @@ export const TransactionReviewModal = (props: TransactionReviewModalProps) => {
     state === TransactionReviewModalState.submit ? 'Review Transaction' : ''
 
   return (
-    <Modal
-      onClose={onCloseModal}
-      isOpen={isOpen}
-      isCentered
-      scrollBehavior='inside'
-    >
-      <ModalOverlay
-        bg='rgba(0, 0, 0, 0.6)'
-        backdropFilter='auto'
-        backdropBlur='8px'
-      />
+    <Modal onClose={onCloseModal} isOpen={isOpen} isCentered>
+      <ModalOverlay className='bg-ic-black bg-opacity-60 backdrop-blur' />
       <ModalContent
-        backgroundColor={backgroundColor}
-        borderColor={styles.border}
-        borderRadius='10'
-        borderStyle='solid'
-        borderWidth='2px'
-        m={['16px', 0]}
+        className={clsx(
+          'border-ic-gray-100  mx-4 my-0 rounded-xl border-[2px]',
+          isDarkMode ? 'review' : '',
+        )}
```

Comment:
Having the dark mode property helps with loading those styles for dark mode only too 👍 

Feel free to suggest a better alternative.